packages/helm-charts/blockscout/templates/grafana-hook.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{{ .Release.Name }}"
```

Comment:
Could this be made more specific? eg with a suffix to make it obvious what this is